core/settings.py

```diff
@@ -11,7 +11,7 @@ def get(self, key, default=None):
         view = window.active_view()
         project_savvy_settings = view.settings().get("GitSavvy", {})
 
-        if key in project_savvy_settings:
+        if project_savvy_settings and key in project_savvy_settings:
```

Comment:
Can also be fixed by doing this on line 12:

```
        project_savvy_settings = view.settings().get("GitSavvy") or {}
```

Not sure which style we prefer

Reply:
I like your suggustion asfaltboy